bucket/oracle-instant-client.json

```diff
@@ -0,0 +1,42 @@
+{
+    "homepage": "https://www.oracle.com/database/technologies/instant-client.html",
+    "description": "Oracle Instant Client enables applications to connect to a local or remote Oracle Database for development and production deployment.",
+    "license": "https://www.oracle.com/downloads/licenses/instant-client-lic.html",
+    "version": "19.3.0.0.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://download.oracle.com/otn_software/nt/instantclient/19300/instantclient-basic-windows.x64-19.3.0.0.0dbru.zip",
+            "hash": "ffceffb0e0024623cbcf5e48409c9c43c36bd117052451cce09faf73325d9931"
+        },
+        "32bit": {
+            "url": "https://download.oracle.com/otn_software/nt/instantclient/19300/instantclient-basic-nt-19.3.0.0.0dbru.zip",
+            "hash": "05d608ce4ee67a3285c4fd4234fb3cfbd5391c48634b10128bea9d8c36cf8456"
+        }
+    },
+    "suggest": {
+       "Microsoft Visual Studio 2017 Redistributable": "vcredist2017"
+    },
+    "extract_dir": "instantclient_19_3",
+    "env_add_path": ".",
+    "pre_install": "mkdir $dir\\network\\admin",
+    "persist": "network\\admin",
+    "env_set": {
+        "TNS_ADMIN": "$persist_dir\\network\\admin"
+    },
+    "checkver": {
+        "url": "https://www.oracle.com/database/technologies/instant-client/winx64-64-downloads.html",
+        "regex": "Version (\\d+\\.\\d+\\.\\d+\\.\\d+\\.\\d+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://download.oracle.com/otn_software/nt/instantclient/$majorVersion$minorVersion$patchVersion$buildVersion/instantclient-basic-windows.x64-$versiondbru.zip",
+                "extract_dir": "instantclient_$majorVersion_$minorVersion"
+            },
+            "32bit": {
+                "url": "https://download.oracle.com/otn_software/nt/instantclient/$majorVersion$minorVersion$patchVersion$buildVersion/instantclient-basic-nt-$versiondbru.zip",
```

Comment:
```suggestion
                "url": "https://download.oracle.com/otn_software/nt/instantclient/$cleanVersion/instantclient-basic-nt-$versiondbru.zip",
```

Reply:
Just to confirm, will the `$cleanVersion` exclude the last `.0` in the version? The Oracle versioning uses a `X.X.X.X.X` versioning syntax, but the URL is actually just `XXXX`. e.g. Version `19.3.0.0.0` has a URL of `19300`.

Reply:
Morons... Keep it as is